writing-visualising/dashboards_rshiny.qmd

```diff
@@ -178,7 +178,7 @@ Public facing DfE Shiny applications are currently published via the DfE Analyti
 
 You need to alert the Statistics Development Team of any new dashboard publication as early in development as possible and keep us updated on the expected publication date so that we can review the dashboard against DfE standards and set up and support on the deployment to ShinyApps.io. In addition, please notify us of any planned data updates or significant functional updates (at least 2 weeks before publication but preferably as soon as you know any updates are required).
 
-All dashboard publications and major updates need authorisation from the relevant G6 or DD and the stats development team (with the former authorisation e-mail being forwarded on to the [Statistics Development Team](mailto:statistics.development@education.gov.uk)).
+All dashboard publications and major updates need authorisation from the relevant G6 or DD and the stats development team (with the former authorisation e-mail being forwarded on to the [explore education statistics platforms team](mailto:explore.statistics@education.gov.uk)).
```

Comment:
Do they need authorisation from my team? 

And should `You need to alert the Statistics Development Team` be me or you?

Reply:
Nope they shouldn't - have switched that text to be my team